setup.py

```diff
@@ -1,35 +1,33 @@
 from setuptools import setup
 
-with open('requirements.txt') as f:
```

Comment:
comment: I applied black on this file as well - please let me know if I should remove that change again :)

Reply:
awesome!